tests/validation/tests/v3_api/test_encryption_secrets.py

```diff
@@ -0,0 +1,41 @@
+from .common import *  # NOQA
+import pytest
+from .test_rke_cluster_provisioning import rke_config
+
+
+def test_create_custom_cluster_with_encryption():
+    # Adds custom encryption to the RKE config
+    secret_encryption_config = {"secrets_encryption_config": {"enabled":
```

Comment:
Can we have all of this declared outside of the test as a global variable?

Reply:
Addressed in changes.
Config has been changed to a global variable.